src/ResourceManager/Network/Commands.Network/VirtualNetwork/Subnet/NewAzureVirtualNetworkSubnetConfigCommand.cs

```diff
@@ -59,6 +60,17 @@ public override void Execute()
                 subnet.RouteTable.Id = this.RouteTableId;
             }
 
+            if (this.PrivateAccessService != null)
+            {
+                subnet.PrivateAccessServices = new List<PSPrivateAccessService>();
+                foreach (var item in this.PrivateAccessService)
+                {
+                    var service = new PSPrivateAccessService();
+                    service.Service = item;
+                    subnet.PrivateAccessServices.Add(service);
+                }
```

Comment:
Set cmdlet is supposed to remove values that are already if $null was passed. I think this make no sense for New i.e. specify parameter with $null just to get no values in resulting object. User could just omit the parameter for this.